Document truly transactional emails v2 feature (#723)
mautic/user-documentation: docs/channels/emails.rst

If set to No, Mautic uses the address and name supplied in the Email 'From' fields.

.. vale off

> 

Author: Added documentation for the `Send to unsubscribed contacts` toggle based on [PR #15995](https://github.com/mautic/mautic/pull/15995) which introduces this feature in the Email Advanced tab. The PR description shows the new toggle allows force-sending emails to unsubscribed contacts for transactional purposes.

Source: https://github.com/mautic/mautic/pull/15995

Send to unsubscribed contacts
*****************************

.. vale on

> The **Send to unsubscribed contacts** toggle allows you to send Emails to Contacts who have unsubscribed from your communications. This feature is available in the Advanced tab for both Template and Segment Emails.

Reviewer: ```suggestion
.. vale off

The **Send to unsubscribed contacts** toggle allows you to send Emails to Contacts who have unsubscribed from your communications. This feature is available in the Advanced tab for both Template and Segment Emails.

.. vale on
```

Use this option for truly transactional communications that Contacts must receive regardless of their subscription status, such as:

- Terms and conditions updates
- Legal notifications
- Account-related communications
- Service announcements

Enabling the toggle
===================

#. Open the Email you want to edit.
#. Click the **Advanced** tab.
#. Set **Send to unsubscribed contacts** to **Yes**.
#. A warning message appears asking you to confirm this action, as sending Emails to unsubscribed Contacts may have legal implications.
#. Confirm to enable the setting, then save the Email.

.. warning::

>     Use this feature responsibly. Sending marketing content to unsubscribed Contacts may violate data protection regulations such as GDPR, CAN-SPAM, or CASL. Only use this setting for legitimate transactional communications.

Reviewer: There are a couple of things here:

- Tidy up the indentation based on our style guide.
- Expand GDPR, CAN-SPAM, and CASL to introduce the abbreviation

```suggestion
.. warning::

   Use this feature responsibly. Sending marketing content to unsubscribed Contacts may violate data protection regulations such as GDPR, CAN-SPAM, or CASL. Only use this setting for legitimate transactional communications.
```

.. note::

>     Enabling this toggle requires the **Send to unsubscribed contacts** permission assigned to your Role. See :doc:`/users_roles/managing_roles` for details on configuring permissions.

Reviewer: A couple of things:

- Tidy up the indentation based on our style guide.
- Wrap the note admonition in vale off/on statements

```suggestion
.. note::

   Enabling this toggle requires the **Send to unsubscribed contacts** permission assigned to your Role. See :doc:`/users_roles/managing_roles` for details on configuring permissions.
```

Frequency rules behavior
========================

> When **Send to unsubscribed contacts** is enabled, the Email doesn't count towards the Contact's :doc:`frequency rules </contacts/frequency_rules>` limits. This ensures that important transactional Emails aren't blocked by communication frequency restrictions.

Reviewer: Change passive to active voice.

If set to No, Mautic uses the address and name supplied in the Email 'From' fields.

.. vale off

Send to unsubscribed contacts
*****************************

The **Send to unsubscribed contacts** toggle allows you to send Emails to Contacts who have unsubscribed from your communications. This feature is available in the Advanced tab for both Template and Segment Emails.

.. vale on

Use this option for truly transactional communications that Contacts must receive regardless of their subscription status, such as:

- Terms and conditions updates
- Legal notifications
- Account-related communications

> - Service announcements

Reviewer: Use asterisks instead of dashes for unordered list.

```suggestion
* Terms and conditions updates
* Legal notifications
* Account-related communications
* Service announcements
```

Enabling the toggle
===================

#. Open the Email you want to edit.
#. Click the **Advanced** tab.
#. Set **Send to unsubscribed contacts** to **Yes**.
#. A warning message appears asking you to confirm this action, as sending Emails to unsubscribed Contacts may have legal implications.
#. Confirm to enable the setting, then save the Email.

.. warning::

>    Use this feature responsibly. Sending marketing content to unsubscribed Contacts may violate data protection regulations such as the General Data Protection Regulation (GDPR), the Controlling the Assault of Non-Solicited Pornography And Marketing Act (CAN-SPAM), or Canada's Anti-Spam Legislation (CASL). Only use this setting for legitimate transactional communications.

Reviewer: ```suggestion
   Use this feature responsibly. Sending marketing content to unsubscribed Contacts may violate data protection regulations such as the General Data Protection Regulation - GDPR, the Controlling the Assault of Non-Solicited Pornography And Marketing Act - CAN-SPAM Act, or Canada's Anti-Spam Legislation - CASL. Only use this setting for legitimate transactional communications.
```

If set to No, Mautic uses the address and name supplied in the Email 'From' fields.

.. vale off

Send to unsubscribed contacts
*****************************

> 

Reviewer: ```suggestion

.. vale on

```

The **Send to unsubscribed contacts** toggle allows you to send Emails to Contacts who have unsubscribed from your communications. This feature is available in the Advanced tab for both Template and Segment Emails.

.. vale on

Use this option for truly transactional communications that Contacts must receive regardless of their subscription status, such as:

* Terms and conditions updates
* Legal notifications
* Account-related communications
* Service announcements

Enabling the toggle
===================

> #. Open the Email you want to edit.
> #. Click the **Advanced** tab.
> #. Set **Send to unsubscribed contacts** to **Yes**.
> #. A warning message appears asking you to confirm this action, as sending Emails to unsubscribed Contacts may have legal implications.
> #. Confirm to enable the setting, then save the Email.

Reviewer: ```suggestion
.. vale off

#. Open the Email you want to edit.
#. Click the **Advanced** tab.
#. Set **Send to unsubscribed contacts** to **Yes**.
#. A warning message appears asking you to confirm this action, as sending Emails to unsubscribed Contacts may have legal implications.
#. Confirm to enable the setting, then save the Email.

.. vale on
```

.. warning::

   **In many countries, sending marketing Emails to Contacts who have unsubscribed is illegal.** This feature exists solely for genuinely transactional communications such as receipts, password resets, legal notices, and account updates—not marketing content.

>    Misusing this feature to send marketing or promotional Emails to unsubscribed Contacts can result in serious legal consequences, including fines and penalties under data protection regulations such as the General Data Protection Regulation - GDPR, the Controlling the Assault of Non-Solicited Pornography And Marketing Act - CAN-SPAM Act, or Canada's Anti-Spam Legislation - CASL.

Reviewer: Add GDPR, CASL, SPAM, and CAN to `accept.txt` file.

.. vale off

.. note::

   Enabling this toggle requires the **Send to unsubscribed contacts** permission assigned to your Role. See :doc:`/users_roles/managing_roles` for details on configuring permissions.

.. vale on

Frequency rules behavior
========================

> Enabling **Send to unsubscribed contacts** prevents the Email from counting towards the Contact's :doc:`frequency rules </contacts/frequency_rules>` limits. This ensures that important transactional Emails reach Contacts regardless of communication frequency restrictions.

Reviewer: ```suggestion
.. vale off

Enabling **Send to unsubscribed contacts** prevents the Email from counting towards the Contact's :doc:`frequency rules </contacts/frequency_rules>` limits. This ensures that important transactional Emails reach Contacts regardless of communication frequency restrictions.

.. vale on
```

If set to No, Mautic uses the address and name supplied in the Email 'From' fields.

.. vale off

> Send to unsubscribed contacts

Reviewer: ```suggestion
.. _send to unsubscribed Contacts

Send to unsubscribed contacts
```

*****************************

> The **Send to unsubscribed contacts** toggle allows you to send Emails to Contacts who have unsubscribed from your communications. This feature is available in the Advanced tab for both Template and Segment Emails.

Reviewer: ```suggestion
The **Send to unsubscribed contacts** toggle allows you to send Emails to Contacts who have unsubscribed from your communications. This feature is available in the **Advanced** tab for both Template and Segment Emails.
```

* Terms and conditions updates
* Legal notifications
* Account-related communications
* Service announcements

Enabling the toggle
===================

.. vale off

#. Open the Email you want to edit.
#. Click the **Advanced** tab.
#. Set **Send to unsubscribed contacts** to **Yes**.
#. A warning message appears asking you to confirm this action, as sending Emails to unsubscribed Contacts may have legal implications.
#. Confirm to enable the setting, then save the Email.

.. vale on

.. warning::

   **In many countries, sending marketing Emails to Contacts who have unsubscribed is illegal.** This feature exists solely for genuinely transactional communications such as receipts, password resets, legal notices, and account updates—not marketing content.

   Misusing this feature to send marketing or promotional Emails to unsubscribed Contacts can result in serious legal consequences, including fines and penalties under data protection regulations such as the General Data Protection Regulation - GDPR, the Controlling the Assault of Non-Solicited Pornography And Marketing Act - CAN-SPAM Act, or Canada's Anti-Spam Legislation - CASL.

.. vale off

.. note::

>    Enabling this toggle requires the **Send to unsubscribed contacts** permission assigned to your Role. See :doc:`/users_roles/managing_roles` for details on configuring permissions.

Reviewer: There are a couple of things here:

- Adjust the wording for clarity.
- Link directly to "Setting granular permissions" section for easy access:

```suggestion
   Enabling this toggle requires the **Send to unsubscribed contacts** Email permission assigned to your Role. See :ref:`Setting Role permissions <setting granular permissions>` for details on configuring permissions.
```

If set to No, Mautic uses the address and name supplied in the Email 'From' fields.

.. vale off

.. _send to unsubscribed Contacts:

Send to unsubscribed contacts
*****************************

The **Send to unsubscribed contacts** toggle allows you to send Emails to Contacts who have unsubscribed from your communications. This feature is available in the **Advanced** tab for both Template and Segment Emails.

.. vale on

Use this option for truly transactional communications that Contacts must receive regardless of their subscription status, such as:

* Terms and conditions updates
* Legal notifications
* Account-related communications
* Service announcements

Enabling the toggle
===================

.. vale off

#. Open the Email you want to edit.
#. Click the **Advanced** tab.
#. Set **Send to unsubscribed contacts** to **Yes**.
#. A warning message appears asking you to confirm this action, as sending Emails to unsubscribed Contacts may have legal implications.
#. Confirm to enable the setting, then save the Email.

> .. vale on
> 
> .. warning::
> 
>    **In many countries, sending marketing Emails to Contacts who have unsubscribed is illegal.** This feature exists solely for genuinely transactional communications such as receipts, password resets, legal notices, and account updates—not marketing content.
> 
>    Misusing this feature to send marketing or promotional Emails to unsubscribed Contacts can result in serious legal consequences, including fines and penalties under data protection regulations such as the General Data Protection Regulation - GDPR, the Controlling the Assault of Non-Solicited Pornography And Marketing Act - CAN-SPAM Act, or Canada's Anti-Spam Legislation - CASL.

Reviewer: There are a couple of things to address Vale warnings:

- Remove 'SPAM' and 'CAN' from `accept.txt` file.
- Apply vale off/on statement to ignore the abbriviations.

 
```suggestion
.. warning::

   **In many countries, sending marketing Emails to Contacts who have unsubscribed is illegal.** This feature exists solely for genuinely transactional communications such as receipts, password resets, legal notices, and account updates—not marketing content.
   
   .. vale off

   Misusing this feature to send marketing or promotional Emails to unsubscribed Contacts can result in serious legal consequences, including fines and penalties under data protection regulations such as the General Data Protection Regulation - GDPR, the Controlling the Assault of Non-Solicited Pornography And Marketing Act - CAN-SPAM Act, or Canada's Anti-Spam Legislation - CASL.
   
   .. vale on
```
